template.rb

```diff
@@ -10,24 +10,17 @@ def apply_template!
   install_graphviz
 
   template 'Gemfile.tt', force: true
-
   template 'README.md.tt', force: true
+  template 'example.env.tt'
+  template 'ruby-version.tt', '.ruby-version', force: true
+
   remove_file 'README.rdoc'
 
-  template 'example.env.tt'
   copy_file 'editorconfig', '.editorconfig'
   copy_file 'rspec', '.rspec'
   copy_file 'gitignore', '.gitignore', force: true
   copy_file 'overcommit.yml', '.overcommit.yml'
-  template 'ruby-version.tt', '.ruby-version', force: true
   copy_file 'simplecov', '.simplecov'
-  copy_file '.erdconfig'
```

Comment:
Why are you deleting `.erdconfig`? I don't see this copy action being moved somewhere else.

Reply:
Probably it was deleted by mistake, after a code conflict fix. Sorry about that.